### PR TITLE
Restructure application into modules

### PR DESCRIPTION
**Pull request by gordon-cheung**

### CODE REVIEW

File: frontend/src/modules/lobby/components/Nickname.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
-  <div>
-    <b-button v-b-modal.modal-2>Nickname</b-button>
+  <span>
```
Comment (bseto) [inline frontend/src/modules/lobby/components/Nickname.vue:2]: Just curious, any reason for this being a span vs div?

File: frontend/src/modules/lobby/view/Lobby.vue [hunk 2]
```diff
@@ -55,9 +58,8 @@ export default {
       this.connectionState = "CONNECTED";
```
Comment (bseto) [inline frontend/src/modules/lobby/view/Lobby.vue:58]: do we even use this connectionState

File: frontend/src/modules/scribble/components/CanvasPanel.vue [hunk 3]
```diff
@@ -10,7 +10,7 @@ import Canvas from "./Canvas.vue";
 import BrushSelector from "./BrushSelector.vue";
 import { createBrushStyle } from "../utility/BrushStyleUtils";
 import { createBrushStrokeMessage } from "../utility/WebSocketMessageUtils";
-import { EventBus } from "../eventBus.js";
+import { EventBus } from "@/eventBus.js";
```
Comment (mpminardi) [inline frontend/src/modules/scribble/components/CanvasPanel.vue:13]: There a reason we are only doing this for some of the imports?

File: frontend/src/modules/lobby/components/Nickname.vue (hunk 1 above)
Comment (gordon-cheung) [reply frontend/src/modules/lobby/components/Nickname.vue:2]: its so the button is in the same line

File: frontend/src/modules/lobby/view/Lobby.vue (hunk 2 above)
Comment (gordon-cheung) [reply frontend/src/modules/lobby/view/Lobby.vue:58]: Uhh nope I'll remove it next time i clean up

File: frontend/src/modules/scribble/components/CanvasPanel.vue (hunk 3 above)
Comment (gordon-cheung) [reply frontend/src/modules/scribble/components/CanvasPanel.vue:13]: Good point, should find a consistent format for these

**Pull request merged**